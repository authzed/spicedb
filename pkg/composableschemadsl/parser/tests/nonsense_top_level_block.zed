nonsense thing {
  relation: user
  permission view = user
}
