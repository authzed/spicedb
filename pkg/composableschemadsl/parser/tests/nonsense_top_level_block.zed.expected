NodeTypeFile
  end-rune = -1
  input-source = top-level block with unrecognized keyword
  start-rune = 0
  child-node =>
    NodeTypeError
      end-rune = -1
      error-message = Unexpected token at root level: TokenTypeIdentifier
      error-source = nonsense
      input-source = top-level block with unrecognized keyword
      start-rune = 0